survey/locale/es/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,119 @@
+# SOME DESCRIPTIVE TITLE.
```

Comment:
I think it might be. There's desktop and web apps people could use to edit .po files, but I'm not sure it'd make it that much easier.